modules/doc/content/getting_started/examples_and_tutorials/tutorial04_meshing/step11_rgmb_het_hom.md

```diff
@@ -0,0 +1,128 @@
+# Reactor Geometry Mesh Builder Example: Conversion of Heterogeneous to Homogeneous Sodium-Cooled Fast Reactor Core Mesh (ABTR)
+
```

Comment:
maybe add a note

this step is Griffin only, the regular MOOSE tutorial ends here

Reply:
Most of this tutorial is MOOSE only (everything up to het mesh generation), only the last step (conversion from het to hom) is Griffin only

Reply:
I agree but we should signal the end of the MOOSE-only content smh

Reply:
I will add a comment about this

Reply:
Added a comment to this file where the endpoint is